lib/config.js

```diff
@@ -8,33 +8,27 @@ const GLOBAL_CONFIG = Symbol('globalConfig');
 const PROJECT_CONFIG = Symbol('projectConfig');
 const LOCAL_CONFIG = Symbol('localConfig');
 
-exports.GLOBAL_CONFIG = GLOBAL_CONFIG;
-exports.PROJECT_CONFIG = PROJECT_CONFIG;
-exports.LOCAL_CONFIG = LOCAL_CONFIG;
-
 function getNcurcPath() {
-  if (process.env.XDG_CONFIG_HOME !== 'undefined' &&
-      process.env.XDG_CONFIG_HOME !== undefined) {
-    return path.join(process.env.XDG_CONFIG_HOME, 'ncurc');
-  } else {
-    return path.join(os.homedir(), '.ncurc');
+  let configHome = os.homedir();
+  if (process.platform === 'linux') {
+    configHome = process.env.XDG_CONFIG_HOME || `${os.homedir()}/.config`;
```

Comment:
I guess this would be a breaking change..maybe it would be safer to check whether `$HOME/.ncurc` exists and emit a warning if it does (also falling back to it) before completely moving to `$HOME/.config/.ncurc` otherwise people might be surprised why their config suddenly stopped working

Reply:
good point - will do!